liota/core/package_manager.py

```diff
@@ -1273,6 +1302,35 @@ def initialize():
             return
     assert(stat.S_ISFIFO(os.stat(package_messenger_pipe).st_mode))
 
+    # Validate package responser pipe
```

Comment:
Please don't replicate 1276-1303 ... make it a function and pass in the path to the fifo you care about.

Reply:
Done. Used a unified function of validate_named_pipe() for all named pipe validation.